experimenter/experimenter/targeting/constants.py

```diff
@@ -1667,34 +1630,38 @@ def __post_init__(self):
     application_choice_names=(Application.DESKTOP.name,),
 )
 
-WHATS_NEW_NOTIFICATION_SIDEBAR_VERTICAL_TABS_ROLLOUT_V2 = NimbusTargetingConfig(
+WHATS_NEW_NOTIFICATION_SIDEBAR_VERTICAL_TABS_ROLLOUT_V3 = NimbusTargetingConfig(
     name=(
         "Windows 10+ users with 1hr+ of inactivity in the past day "
-        "who are running a background task and are "
-        "not enrolled in treatment-a of WNN sidebar/vertical tabs experiment"
+        "who are running a background task and have CFR pref enabled "
+        "and excludes users with enterprise policies, "
+        "enrolled in treatment-a of WNN sidebar/vertical tabs experiment, "
+        "and enrolled in the previous rollout of WNN sidebar/vertical tabs "
     ),
-    slug="whats_new_notification_sidebar_vertical_tabs_rollout_v2",
+    slug="whats_new_notification_sidebar_vertical_tabs_rollout_v3",
     description=(
         "Windows 10+ users with 1hr+ of inactivity in the past day "
-        "who are running a background task and are "
-        "not enrolled in treatment-a of WNN sidebar/vertical tabs experiment"
+        "who are running a background task and have CFR pref enabled "
+        "and not enrolled in treatment-a of WNN sidebar/vertical tabs "
+        "experiment and rollout"
     ),
     targeting="""
     (
-        (
-            os.isWindows
-            &&
-            (os.windowsVersion >= 10)
-        )
+        (os.isWindows && (os.windowsVersion >= 10))
         &&
-        (
-            ((currentDate|date - defaultProfile.currentDate|date) / 3600000 >= 1)
-        )
+        (((currentDate|date - defaultProfile.currentDate|date) / 3600000 >= 1))
```

Comment:
Theres an extra pair
```suggestion
        ((currentDate|date - defaultProfile.currentDate|date) / 3600000 >= 1)
```